brave/brave-core
# Adding option to remove sites from auto-contribute

Addresses: https://github.com/brave/brave-browser/issues/967

Native ledger work:
https://github.com/brave-intl/bat-native-ledger/pull/87
https://github.com/brave-intl/bat-native-ledger/pull/93

## Submitter Checklist:


## Test Plan:
1. Launch Brave, enable rewards
2. Visit 6 sites to populate contributions table
3. In the contribution table, click `See all ${n} sites` to display the removal modal
4. Click the `x` icon next to a few publishers
5. Confirm they are excluded and removed from the contribution table

## Reviewer Checklist:
